### PR TITLE
fix: Resize rich text properly (WEBAPP-5515)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/view_model/content/InputBarViewModel.js [hunk 1]
```diff
@@ -153,6 +153,10 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
         .join('');
     });
 
+    this.richTextInput.subscribe(() => {
+      $('.shadow-input').trigger('input');
+    });
```
Comment (gregor) [inline app/script/view_model/content/InputBarViewModel.js]: Could be single line

File: app/page/template/content/conversation/input-bar.htm [hunk 2]
```diff
@@ -29,7 +29,7 @@
                   data-uie-name="input-message"
                   dir="auto">
         </textarea>
-        <div class="shadow-input" data-bind="html: richTextInput, scrollSync: '#conversation-input-bar-text', resize: input" data-uie-name="input-message-rich-text"></div>
+        <div class="shadow-input" data-bind="html: richTextInput, scrollSync: '#conversation-input-bar-text', resize: input, resize_callback: scrollMessageList" data-uie-name="input-message-rich-text"></div>
```
Comment (bennycode) [inline app/page/template/content/conversation/input-bar.htm]: `resize_callback` -> `resizeCallback` if possible

File: app/script/view_model/content/InputBarViewModel.js (hunk 1 above)
Comment (atomrc) [reply app/script/view_model/content/InputBarViewModel.js]: Let's go for this solution for now. But please can we try to remove DOM querying and jQuery in the near future?

File: app/page/template/content/conversation/input-bar.htm (hunk 2 above)
Comment (gregor) [reply app/page/template/content/conversation/input-bar.htm]: Then we have to rename in other places as well. This was missed during the renaming of binding and is decared in `CommonBindings.js`

**Comment by gregor** [comment]
LGTM

File: app/script/view_model/content/InputBarViewModel.js (hunk 1 above)
Comment (AndyLnd) [reply app/script/view_model/content/InputBarViewModel.js]: I'm totally with you. This is a real band aid solution.

**Pull request merged**